setup.py

```diff
@@ -4,43 +4,20 @@
 Setup script for pipe-orbcomm
 """
 
-import codecs
-import os
-
-import setuptools
 from setuptools import find_packages
 from setuptools import setup
 
-import subprocess
-from distutils.command.build import build as _build
-
-
-package = __import__('pipe_events')
-
-
-DEPENDENCIES = [
-    "jinja2-cli",
-]
-
-with codecs.open('README.md', encoding='utf-8') as f:
-    readme = f.read().strip()
-
-with codecs.open('requirements.txt', encoding='utf-8') as f:
-    DEPENDENCY_LINKS=[line for line in f]
-
 setup(
-    author=package.__author__,
-    author_email=package.__email__,
-    description=package.__doc__.strip(),
-    include_package_data=True,
-    install_requires=DEPENDENCIES,
-    license="Apache 2.0",
-    long_description=readme,
     name='pipe-events',
+    version='4.1.2',
```

Comment:
only thing I would add if possible the new tag you create, to be consistent. I saw that the last was the `4.2.0`.